CONTRIBUTING.md

```diff
@@ -358,7 +362,24 @@ you just need to update the reference images as follows:
     new tests should now pass.
 
 Note: If you have run `test/ipynb/mpl_tester.ipynb` locally it is possible some file metadata has changed, **please do not commit and push changes to this file unless they were intentional**.
-### Development Cycle
+
+## Style and lint
+
+Qiskit Terra uses 2 tools for verify code formatting and lint checking. The
```

Comment:
I'd rather phase out the name Terra from docs that users see - just saying Qiskit (especially as black will likely be done in other repos too).

Reply:
We can clean this up later, I was planning to update the overall contributing documentation when we've unified on using black everywhere: https://qiskit.org/documentation/contributing_to_qiskit.html#style-guide right now it just says pycodestyle and pylint